vendor.go

```diff
@@ -131,16 +131,12 @@ func vendorPackages(depsGoHome string, projectdir string) {
 		if info == nil {
 			return nil
 		}
-		if info.IsDir() {
-			return nil
-		}
-		// Not sure this works, but seems feasible to ignore
-		// vendoring from libs, they should not do that
-		if strings.Contains(path, "/vendor/") {
-			return nil
+
+		if ignore, err := ignorePath(path, info.IsDir()); ignore {
+			return err
 		}
 
-		if !strings.HasSuffix(path, ".go") {
+		if info.IsDir() {
 			return nil
 		}
 
```

Comment:
The strings.HasSuffix check for the _test files could also be part of the ignorePath logic no ?

Reply:
yes, I forgot it. Thanks